src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -493,7 +527,17 @@ function ChartDeploymentHistory({
                         </div>
                     </div>
                 )}
-                {(selectedDeploymentTabName === DEPLOYMENT_HISTORY_TAB.HELM_GENERATED_MANIFEST || selectedDeploymentTabName === DEPLOYMENT_HISTORY_TAB.VALUES_YAML ) && renderCodeEditor()}
+                {(selectedDeploymentTabName === DEPLOYMENT_HISTORY_TAB.HELM_GENERATED_MANIFEST ||
+                    selectedDeploymentTabName === DEPLOYMENT_HISTORY_TAB.VALUES_YAML) &&
+                    renderCodeEditor()}
+                {selectedDeploymentTabName === DEPLOYMENT_HISTORY_TAB.ARTIFACTS && (
+                    <VirtualHistoryArtifact
+                        titleName={'titleNmae'}
```

Comment:
fix the titleNmae to titleName. Will it be hardcoded value? if yes, we can directly write it as titleName="titleName"

Reply:
we are giving dynamic value now